### PR TITLE
HPCC-26119 Update Environment.conf documentation

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful integration testing: 
http://10.224.20.18/view/Docs-gp/job/Doc-Build-04-GPRepo2/

### CODE REVIEW

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 1]
```diff
@@ -1131,6 +1131,55 @@ thor:
         <para>To add environment values you can insert them into your
         customization configuration YAML file when you deploy your
         containerized HPCC Systems.</para>
+
+        <sect4>
+          <title>Bare Metal Variables for Cloud </title>
+
+          <para>There are several environment.conf settings for Bare Metal
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1138]: bare-metal is hyphenated elsewhere in this manual

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 2]
```diff
@@ -1131,6 +1131,55 @@ thor:
...
+          systems which are not valid for cloud environments. Some of those
+          variables are available for container and cloud deployments and can
+          be set using the helm chart. The following bare metal
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1141]: Helm chart 
(Helm s/b capitalized)

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 3]
```diff
@@ -1131,6 +1131,55 @@ thor:
...
+          environment.conf values have these equivalent values which can be
+          set for containerized instances. </para>
+
+          <para><informaltable>
+              <tgroup cols="2">
+                <tbody>
+                  <row>
+                    <entry><emphasis role="bold">Environment.conf
+                    Vallue</emphasis></entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1150]: typo Value

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

**Review by JamesDeFabia — APPROVED** [review]
Looks ok from my POV, but a SME needs to review

**Comment by ghalliday** [comment]
@g-pan waiting for your response to Jake....

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 4]
```diff
@@ -1177,10 +1177,16 @@
     <sect1 id="SysAdm_Env.conf">
       <title>Environment.conf</title>
 
-      <para>Another component of HPCC Systems configuration is the
+      <para>A component of HPCC Systems on bare metal configuration is the
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:1180]: bare-metal

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 5]
```diff
@@ -1177,10 +1177,16 @@
...
       environment.conf file. Environment.conf contains some global definitions
       that the configuration manager uses to configure the HPCC Systems. In
-      most cases, the defaults are sufficient.</para>
+      most cases, the defaults are sufficient. </para>
+
+      <para>The environment.conf file only works for bare metal deployments.
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:1185]: bare-metal

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
A couple of minor issues (comments inline)

**Review by jakesmith — APPROVED** [review]
@g-pan - looks fine

**Review by JamesDeFabia — APPROVED** [review]
Good from my POV

**Comment by g-pan** [comment]
squashed and ready to merge

**Pull request merged**